### PR TITLE
5.11 - merge in security

### PR DESCRIPTION
**Pull request by seamuslee001**
ping @eileenmcnaughton

### CODE REVIEW

File: CRM/Contact/BAO/Query.php [hunk 1]
```diff
@@ -3173,21 +3176,22 @@ public function tagSearch(&$values) {
...
     $op = "LIKE";
     $value = "%{$value}%";
+    $escapedValue = CRM_Utils_Type::escape("%{$value}%", 'String');
 
     $useAllTagTypes = $this->getWhereValues('all_tag_types', $grouping);
     $tagTypesText = $this->getWhereValues('tag_types_text', $grouping);
 
-    $etTable = "`civicrm_entity_tag-" . $value . "`";
-    $tTable = "`civicrm_tag-" . $value . "`";
+    $etTable = "`civicrm_entity_tag-" . uniqid() . "`";
```
Comment (eileenmcnaughton) [inline CRM/Contact/BAO/Query.php]: @seamuslee001 maybe the group one needs to look more like this?

Comment (seamuslee001) [reply CRM/Contact/BAO/Query.php]: @eileenmcnaughton that is exactly my thinking :-)

Comment (eileenmcnaughton) [reply CRM/Contact/BAO/Query.php]: were you also thinking the other thing I'm thinking?

Comment (seamuslee001) [reply CRM/Contact/BAO/Query.php]: I'm not sure B1

Comment (eileenmcnaughton) [reply CRM/Contact/BAO/Query.php]: :-)

**Pull request merged**